examples/dynamicWhitelisting.ts

```diff
@@ -0,0 +1,40 @@
+import { PolymathAPI, TransactionParams, BigNumber } from '../src';
+
+/**
+ * Mints the token to a specific investor adding it to whitelist dynamically.
+ * It requires to be called by token owner and a valid security token and no other transfer restrictions impeding the issuance.
+ * @param polymathAPI Reference to polymathAPI instance.
+ * @param ticker Ticker of the Security Token
+ * @param issueMultiParams Transaction params for issuing to multiple investors.
+ */
+export const issueTokenToNonWhitelistedInvestor = async (
+  polymathAPI: PolymathAPI,
+  ticker: string,
+  signTransferData: TransactionParams.SecurityToken.SignTransferData,
+  amount: BigNumber,
+) => {
+  // Create a Security Token Instance
+  const securityTokenInstance = await polymathAPI.tokenFactory.getSecurityTokenInstanceFromTicker(ticker!);
+
+  // SignTransferData example
+  //   {
+  //     investorsData: [
+  //       {
+  //         investorAddress: '0xbfd02b95a0ba36807902d1c31b66edae90ec2151',
+  //         canReceiveAfter: new Date(2019, 11, 1),
+  //         canSendAfter: new Date(2019, 11, 1),
+  //         expiryTime: new Date(2020, 12, 1),
```

Comment:
This date is meant to be in the future, in these cases I usually use 2030 as it’s likely this project will still be maintained for several years

Reply:
Yeah, makes sense. But it's only a example of the data, it's not really using it.